src/test/java/wedlog/address/logic/commands/VendorAddCommandIntegrationTest.java

```diff
@@ -37,6 +40,28 @@ public void execute_newVendor_success() {
         assertCommandSuccess(new VendorAddCommand(validVendor), model,
                 String.format(VendorAddCommand.MESSAGE_SUCCESS, Messages.format(validVendor)),
                 expectedModel);
+
+        Vendor insertedVendor = model.getFilteredVendorList().get(0);
+
+        // Check that all fields of the Vendor have been stored properly in WedLog
+        assertEquals(insertedVendor.getName(), validVendor.getName());
+        assertEquals(insertedVendor.getPhone(), validVendor.getPhone());
+        assertEquals(insertedVendor.getEmail(), validVendor.getEmail());
+        assertEquals(insertedVendor.getAddress(), validVendor.getAddress());
+        assertEquals(insertedVendor.getTags(), validVendor.getTags());
```

Comment:
Is this line necessary? `assertCommandSuccess` already checks the `model` against the `expectedModel`, which checks between the `ModelManager` objects, which checks between the `AddressBook` objects, and eventually checks between the fields of each `Vendor`.
```suggestion
```